src/xdist/remote.py

```diff
@@ -242,6 +242,7 @@ def setup_config(config, basetemp):
     os.environ["PYTEST_XDIST_WORKER_COUNT"] = str(workerinput["workercount"])
 
     if hasattr(Config, "InvocationParams"):
+        args = list(args) + ["-pno:terminal"]
```

Comment:
This is required because now output in worker processes actually goes to stdout, so `print()` and `-s` works as expected in this setup.